examples/join-clause/join_clause.md

```diff
@@ -1,8 +1,6 @@
 # Join clause
 
-`Query` can take advantage of `table` keys by using a `join clause`. Performs an `inner equijoin`.
-The result is similar to using nested `from clause` and `where clause`. It is implemented as
-a hash join: `table` keys allow you to avoid building a hash table. The type to join on must be `anydata`.
+A query expression can take advantage of keys of a table by using a `join` clause. It performs an inner equijoin. The result is similar to using nested `from` clauses and `where` clause. It is implemented as a hash join. Table keys allow you to avoid building a hash table. The type to join on must be `anydata`.
```

Comment:
```suggestion
A query expression can take advantage of keys of a table by using a `join` clause. It performs an inner equijoin. The result is similar to using nested `from` clauses and `where` clause. It is implemented as a hash join. Table keys allow you to avoid building a hash table. The type to join must be `anydata`.
```